DependencyInjection/CmfCreateExtension.php

```diff
@@ -86,20 +86,25 @@ public function load(array $configs, ContainerBuilder $container)
 
         $this->loadSecurity($config['security'], $loader, $container);
 
-        $hasMapper = false;
         if ($config['persistence']['phpcr']['enabled']) {
             $this->loadPhpcr($config['persistence']['phpcr'], $loader, $container);
-            $hasMapper = true;
         } else {
             // TODO: we should leverage the mediabundle here and not depend on phpcr
             $container->setParameter($this->getAlias() . '.image_enabled', false);
         }
+        if ($config['persistence']['orm']['enabled']) {
+            $this->loadOrm($config['persistence']['orm'], $loader, $container);
+        }
+        $hasMapper = false;
         if (isset($config['object_mapper_service_id'])) {
             $container->setAlias($this->getAlias() . '.object_mapper', $config['object_mapper_service_id']);
             $hasMapper = true;
+        } else {
+            $container->setAlias($this->getAlias() . '.object_mapper', $this->getAlias() . '.chain_mapper');
+            $hasMapper = count($container->findTaggedServiceIds('cmf_create.mapper')) > 0;
```

Comment:
i think this is not correct. the idea of the tags is that the order in which bundles are loaded does not matter. if i neither enable the default phpcr nor orm mapper, but provide my custom mapper, it might not be known at this point.